templates/checks.json

```diff
@@ -0,0 +1,11 @@
+{
+  "checks": { {% for check in checks %}
```

Comment:
Template render uses the Jinja2 template render. And with better formatting it added those newlines in the generated template resulting in Sensu being unable to read the jsons